tests/ci/codebuild/run_accp_overkill.yml

```diff
@@ -7,3 +7,14 @@ phases:
   build:
     commands:
       - ./tests/ci/run_accp_overkill.sh
+reports:
+  unit-tests:
+    files:
+      - 'build/reports/unit-tests/**'
```

Comment:
overkill/dieharder tests generate JUnit reports?

Reply:
Looks like I'm wrong on this and will remove it. We should figure out why at some point.